dr-secondary-cluster.md

```diff
@@ -0,0 +1,410 @@
+    ```
+
+    ```
+    +----------------------+----------+--------------------+---------------------+---------------------+
+    | Destination          | Size     | BackupTS           | Queue Time          | Execution Time      |
+    +----------------------+----------+--------------------+---------------------+---------------------+
+    | s3://backup          | 10315858 | 431434047157698561 | 2022-02-25 19:57:59 | 2022-02-25 19:57:59 |
+    +----------------------+----------+--------------------+---------------------+---------------------+
+    1 row in set (2.11 sec)
+    ```
+
+    After the `BACKUP` command is executed, TiDB returns metadata about the backup data. Pay attention to `BackupTS`, because data generated before it is backed up. In this document, we use `BackupTS` as **the end of data check** and **the start of incremental migration scanning by TiCDC**.
+
+3. Restore data. Run the `RESTORE` command in the secondary cluster to restore data:
+
+    ```sql
+    mysql> RESTORE DATABASE * FROM '`s3://backup?access-key=minio&secret-access-key=miniostorage&endpoint=http://10.0.1.10:6060&force-path-style=true`';
+    ```
+
+    ```
+    +----------------------+----------+----------+---------------------+---------------------+
+    | Destination          | Size     | BackupTS | Queue Time          | Execution Time      |
+    +----------------------+----------+----------+---------------------+---------------------+
+    | s3://backup          | 10315858 | 0        | 2022-02-25 20:03:59 | 2022-02-25 20:03:59 |
+    +----------------------+----------+----------+---------------------+---------------------+
+    1 row in set (41.85 sec)
+    ```
+
+#### Replicate incremental data
+
+After migrating data as described in the preceding section, you can replicate incremental data from the primary cluster to the secondary cluster starting from the **BackupTS**.
+
+1. Create a changefeed.
+
+    Create a changefeed configuration file `changefeed.toml`.
+
+    ```toml
+    [consistent]
+    # eventual consistency: Redo logs are used to ensure the eventual consistency in disaster scenarios.
+    level = "eventual"
+    # The size of a single redo log, in MiB. The default value is 64, and the recommended value is less than 128.
+    max-log-size = 64
+    # Interval for refreshing or uploading redo logs to Amazon S3, in milliseconds. The default value is 1000, and the recommended value range is 500-2000.
+    flush-interval = 2000
+    # The path where redo logs are saved.
+    storage = "s3://redo?access-key=minio&secret-access-key=miniostorage&endpoint=http://10.0.1.10:6060&force-path-style=true"
+    ```
+
+    In the primary cluster, run the following command to create a changefeed from the primary to the secondary clusters:
+
+    ```shell
+    tiup cdc cli changefeed create --server=http://10.1.1.9:8300 --sink-uri="mysql://{username}:{password}@10.1.1.4:4000" --changefeed-id="dr-primary-to-secondary" --start-ts="431434047157698561"
```

Comment:
This could make it more readable

```suggestion
    tiup cdc cli changefeed create --server=http://10.1.1.9:8300 \
    --sink-uri="mysql://{username}:{password}@10.1.1.4:4000" \
    --changefeed-id="dr-primary-to-secondary" --start-ts="431434047157698561"
```

Reply:
@allengaoo PTAL